lib/shopify_app/in_memory_session_store.rb

```diff
@@ -1,25 +1,27 @@
 # WARNING - This really only works for development, see README for more details
-class InMemorySessionStore
-  class EnvironmentError < StandardError; end
+module ShopifyApp
+  class InMemorySessionStore
+    class EnvironmentError < StandardError; end
 
-  def self.retrieve(id)
-    repo[id]
-  end
+    def self.retrieve(id)
+      repo[id]
+    end
 
-  def self.store(session)
-    id = SecureRandom.uuid
-    repo[id] = session
-    id
-  end
+    def self.store(session)
+      id = SecureRandom.uuid
+      repo[id] = session
+      id
+    end
 
-  def self.clear
-    @@repo = nil
-  end
+    def self.clear
+      @@repo = nil
+    end
 
-  def self.repo
-    if Rails.env.production?
-      raise EnvironmentError.new("Cannot use InMemorySessionStore in a Production environment")
+    def self.repo
+      if Rails.env.production?
+        raise EnvironmentError.new("Cannot use InMemorySessionStore in a Production environment")
```

Comment:
I really like that we explicitly raise this error here. When people see this error during a deploy, they'll think about what they need to do to fix this. We can make this error more actionable by changing it to:

```
raise EnvironmentError.new(
  "Cannot use InMemorySessionStore in a Production environment . Please initialize ShopifyApp with a model that can store and retrieve sessions"
)
```
We can also add a link to relevant documentation in that error. Maybe a link to a wiki page.